pkg/config/config_windows.go

```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+)
+
+// getConfigDir returns the config directory for the current OS.
+// On Windows, it is at the same location as the executable, by default `C:\Program Files(x86)\MassaStation`.
+func getConfigDir() (string, error) {
+	ex, err := os.Executable()
+	if err != nil {
+		return "", fmt.Errorf("getting executable path: %w", err)
+	}
+
+	dir := filepath.Dir(ex)
+
+	return dir, nil
+}
+
+// getCertDir returns the cert directory for the current OS.
+// On Windows, it is at the same location as the executable, by default `C:\Program Files(x86)\MassaStation\certs`.
+func getCertDir() (string, error) {
+	confDir, err := getConfigDir()
+	if err != nil {
+		return "", err
```

Comment:
better to be consistent in the error messages.

```suggestion
		return "", fmt.Errorf("getting executable path: %w", err)
```

Reply:
We are not trying to get the executable path in that case. But yes, we need to add some information about the fact that we failed to get the certs folder path.